Customizing/global/plugins/Services/Repository/RepositoryObject/ReportEduBio/classes/class.ilObjReportEduBio.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+require_once 'Customizing/global/plugins/Services/Cron/CronHook/ReportMaster/classes/ReportBase/class.ilObjReportBase.php';
+require_once 'Services/GEV/Utils/classes/class.gevUserUtils.php';
+require_once 'Services/GEV/Utils/classes/class.gevSettings.php';
+
+ini_set("memory_limit","2048M"); 
+ini_set('max_execution_time', 0);
+set_time_limit(0);
+
+class ilObjReportEduBio extends ilObjReportBase {
+	protected $relevant_parameters = array();
+
+	public function __construct($a_ref_id = 0) {
+		parent::__construct($a_ref_id);
+	}
+
+	public function initType() {
+		 $this->setType("xreb");
+	}
+
+	protected function createLocalReportSettings() {
+		$this->local_report_settings =
+			$this->s_f->reportSettings('rep_robj_reb');
+	}
+
+	public function prepareRelevantParameters() {
+		$self_id = $this->user_utils->getId();
+		$target_user_id = $_POST["target_user_id"]
+					  ? $_POST["target_user_id"]
+					  : ( $_GET["target_user_id"]
+					  	? $_GET["target_user_id"]
+					  	: $self_id
```

Comment:
Ist der switch zwischen get und post hier nötig? Soweit ich das in Erinnerung hab, steht die usr id doch immer im get, egal ob ich filter oder nicht.